### DISCUSSION TITLE
Can not import document file which has image inside

### DISCUSSION THREAD

**Discussion by nh4ttruong** [outline/outline#6129] [Self Hosting]
Hi team, I tried to import document file which has image inside into my Outline but it seems to not attach any image from original document file. Could I miss something in release note?

**To Reproduce**
1. Go to Outline collection
2. Click on 'Import document' and pick `.doc` file
3. Scroll down to new document and can not see any image inside original document file

**Expected behavior**
A Outline document like original document

**Outline (please complete the following information):**
- Install:  Self-Hosted
- Version: v0.72.2

**Comment by nh4ttruong** [comment]
I convert my `doc` to `docx` and it worked 😄

**Comment by nh4ttruong** [comment]
Sorry, I found new issue relate it. When import a document, my image is shown as `C:`:
![image](https://github.com/outline/outline/assets/61237126/b3e56ec8-e674-40f6-b753-759970b65e1d)
And in network tab, I see that Outline not correctly import into my bucket, so it cannot load image from bucket while images imported by Copy Paste or manually import:
![image](https://github.com/outline/outline/assets/61237126/d0ff00ec-bb50-4c6c-a2fb-75113bd7ea48)

**Reply by tommoor** [reply]
This is likely DNS where the server cannot reach your storage server, please search the discussions for this as it has come up many times.

If you really think it's an issue with the image being in document (this should be supported) then you can upload the docx and I can test separately.
